src/test/java/api/loans/ClaimItemReturnedAPITests.java

```diff
@@ -64,6 +66,21 @@ void canClaimItemReturnedWithComment() {
     assertLoanAndItem(response, comment, dateTime);
   }
 
+  @Test
+  void claimItemReturnFailsWhenEventPublishingFailsWithBadRequestError() {
+    final String comment = "testing";
+    final ZonedDateTime dateTime = getZonedDateTime();
```

Comment:
you don't have to create the variables since you use it only once

Reply:
fixed